.github/workflows/test_fast.yml

```diff
@@ -35,20 +35,6 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Brew Install
-        if: startsWith(matrix.os, 'macos')
-        run: |
-          # speed up install (https://docs.brew.sh/Manpage#environment)
-          export HOMEBREW_NO_AUTO_UPDATE=1 HOMEBREW_NO_INSTALL_CLEANUP=1
-          echo "[command]brew update"
-          brew update
-          echo "[command]brew install ..."
-          brew install bash coreutils
-          # add GNU coreutils and sed to the user PATH
-          # (see instructions in brew install output)
-          echo "$(brew --prefix)/opt/coreutils/libexec/gnubin" \
-            >> "${GITHUB_PATH}"
-
```

Comment:
I find this surprising, Mac OS ships with the BSD coreutils, not the GNU ones, they are similar but have some different CLI options. Cylc has been written into the GNU coreutils and makes use of some features which are not POSIX. If the tests pass with the BSD coreutils (note we don't currently run all tests on Mac OS anyway) then that suggests a lack of test coverage rather than a lack of requirement.

Reply:
Oh, dammit, sorry, ignore me, this is the fast tests not the functional ones, derp.

Ok, yes I think this makes sense, if a test fails because of this removal, then it's really a functional test which relies on the interaction of multiple systems (e.g. GNU coreutils) so should not be tested in this way!